.github/workflows/cicd.yml

```diff
@@ -72,6 +72,25 @@ jobs:
       configurations: '["NativeOnlyDebug", "NativeOnlyRelease"]'
       build_options: /p:BuildOneBranch='True' /t:tools\onebranch
       solution_file: "ebpf-for-windows.sln"
+      architecture: "x64"
+      download_demo_repository: false
+
+  onebranch_ARM64:
```

Comment:
can we use an architecture matrix instead of replicating the job?

Reply:
Seems to work.